R/run-region-updates.R

```diff
@@ -200,16 +205,16 @@ if (sys.nframe() == 0) {
   args <- rru_cli_interface()
   setup_log_from_args(args)
   # tryCatch(withCallingHandlers(
-    run_regional_updates(datasets = datasets, args = args)
-    # ,
-    #                            warning = function(w) {
-    #                              futile.logger::flog.warn(w)
-    #                              rlang::cnd_muffle(w)
-    #                            },
-    #                            error = function(e) {
-    #                              futile.logger::flog.error(capture.output(rlang::trace_back()))
-    #                            }),
-    #        error = function(e) {
-    #          futile.logger::flog.error(e)
-    #        })
+  run_regional_updates(datasets = datasets, args = args)
```

Comment:
Not for this PR but this hack job was from a quick fix from me whilst you were on holiday and means we throw away errors (oops).

Reply:
I had spotted... I'm sure there was a reason but we should reintroduce it. Perhaps the ftry method would be better?

Reply:
it was the issue of an object being passed causing a silent immediate crash - a slight nightmare to debug! Yes `ftry` approach seems sensible.

Reply:
raised #70  so it doesn't get forgotten